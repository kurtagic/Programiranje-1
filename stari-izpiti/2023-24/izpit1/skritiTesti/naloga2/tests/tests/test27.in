2 10 74 617
1 27 571
5 27 365
2 65 443
2 65 389
2 27 468
1 27 271
0 68 105
0 65 498
6 65 262
2 65 117
5 30 708
1 30 594
6 68 154
2 27 166
4 68 142
5 27 571
2 18 24
8 68 583
6 44 646
6 44 201
4 65 97
1 18 71
5 65 406
4 23 392
8 27 305
4 39 352
0 65 502
0 18 36
8 27 496
0 18 6
0 18 442
8 6 691
5 18 85
6 18 609
8 30 529
4 25 70
4 48 275
6 44 696
8 18 328
6 56 541
4 44 522
0 23 312
4 10 123
1 48 552
5 56 236
8 9 579
4 30 541
1 30 148
1 18 702
8 39 265
1 30 396
0 27 706
6 66 196
4 69 6
8 23 368
0 44 216
1 27 576
0 49 530
6 21 422
8 41 602
0 21 380
4 6 190
2 18 620
8 66 60
2 44 107
8 23 472
1 49 527
0 56 522
5 56 38
5 56 446
4 66 163
0 18 88
8 18 395
8 58 331
2 23 507
1 15 41
8 68 556
8 25 6
7 2 252
4 66 488
4 71 594
1 23 665
1 50 342
8 25 211
0 67 710
0 71 364
1 70 39
7 68 700
1 9 51
1 43 380
6 70 44
1 39 631
8 43 489
4 49 622
4 21 107
1 26 319
6 49 540
1 39 421
4 49 317
0 66 198
0 33 62
0 67 417
0 21 702
1 42 542
1 12 698
1 15 418
6 9 122
1 26 20
4 21 579
0 67 476
0 49 645
1 2 403
8 7 349
0 10 666
1 18 111
0 68 233
6 65 670
1 39 425
8 48 319
9 2 197
6 10 113
2 6 607
0 70 285
8 44 559
9 58 248
2 65 434
9 50 695
9 39 258
6 66 415
4 37 343
1 49 435
4 2 306
6 15 618
2 67 482
6 65 614
8 58 511
7 62 567
8 15 531
6 67 371
1 18 458
5 30 28
2 5 635
2 13 560
1 39 83
5 18 598
5 50 708
0 30 402
0 23 184
7 22 186
4 40 276
8 33 11
1 12 121
6 27 491
1 45 481
9 12 10
9 67 97
9 70 655
9 66 6
2 66 46
1 9 20
1 67 126
6 46 448
2 66 177
0 6 345
5 44 477
1 65 473
7 19 227
9 13 619
7 43 401
6 69 360
4 58 304
0 67 321
4 43 346
9 39 355
7 9 553
2 10 365
0 65 387
8 69 376
6 30 618
7 44 670
5 50 23
1 27 464
0 65 365
4 58 559
7 41 51
1 15 320
0 41 288
4 30 112
9 39 298
7 66 506
2 15 552
9 69 10
5 70 384
3 30 543
9 48 582
0 50 185
4 8 231
1 71 457
9 23 285
1 48 339
2 58 593
5 43 147
6 66 501
5 61 690
9 66 142
4 67 103
9 41 1
0 6 479
0 21 207
8 66 60
2 18 603
5 25 459
2 67 291
3 67 707
0 40 471
2 10 176
2 21 471
8 8 595
0 10 281
0 13 540
6 19 477
2 70 645
6 49 141
1 40 451
6 33 201
5 50 207
4 30 136
7 25 653
7 70 243
0 66 183
7 71 35
9 57 295
8 18 478
0 53 338
7 31 195
6 66 458
3 2 520
1 67 9
1 18 157
7 33 570
6 57 398
8 65 81
7 39 492
0 15 28
9 48 453
6 43 82
2 41 545
8 46 354
7 70 109
7 46 477
8 13 339
9 46 357
9 48 66
2 9 340
6 39 560
2 22 147
8 18 549
6 49 672
6 30 332
5 67 71
0 39 606
8 15 393
2 40 630
9 40 651
9 66 524
4 2 546
0 34 321
5 22 545
7 69 526
5 62 438
6 18 89
6 30 513
1 34 402
1 37 414
8 27 690
8 66 9
6 70 271
7 67 690
7 11 16
1 4 604
0 10 155
7 1 427
7 50 570
3 8 124
5 40 560
5 53 571
4 66 685
3 70 521
2 25 346
5 48 80
1 62 689
6 56 537
8 50 678
6 15 76
1 27 94
5 9 44
2 43 506
5 61 341
8 21 224
5 18 454
0 61 471
2 1 529
9 43 89
1 1 187
8 57 482
6 18 90
6 11 431
8 11 309
0 10 398
2 33 60
6 53 384
5 65 641
9 11 603
4 10 400
0 70 121
2 8 608
8 48 218
7 66 518
0 22 362
0 7 386
4 37 490
3 2 595
0 36 43
7 42 387
4 25 621
0 71 306
1 11 473
8 11 68
7 10 4
9 48 187
0 41 385
7 66 4
0 39 344
5 11 244
9 48 240
3 9 679
0 27 260
6 7 388
0 1 276
4 5 468
2 30 546
4 8 254
8 39 120
9 22 476
3 6 373
9 50 381
5 13 463
2 12 23
7 2 78
7 15 1
4 66 214
0 1 698
1 7 469
9 10 197
0 48 520
5 56 109
9 36 285
5 18 690
6 26 613
7 43 148
5 25 110
4 50 552
6 13 442
9 67 53
8 15 54
1 6 165
6 57 525
2 10 113
6 15 347
8 42 309
2 11 553
7 15 338
1 8 188
5 38 40
0 23 309
3 66 683
1 34 683
8 10 56
9 10 167
9 30 325
9 35 108
9 10 346
7 1 469
0 23 220
0 11 403
7 67 645
5 66 690
7 24 125
3 67 35
9 41 242
6 45 323
7 49 208
7 29 218
5 2 344
6 30 533
7 12 354
4 4 516
0 2 411
3 21 152
4 2 530
2 11 651
0 65 657
9 22 479
0 15 575
9 38 587
1 51 468
9 1 632
4 39 337
0 48 69
1 1 33
1 12 422
4 45 62
4 56 256
4 54 512
7 56 102
2 34 136
4 39 436
6 6 122
1 67 110
1 28 128
5 22 586
5 26 295
7 60 304
0 36 489
1 19 220
9 39 128
5 10 625
2 50 356
7 25 304
9 30 136
8 70 371
1 9 310
3 33 249
0 20 678
7 45 429
5 1 372
0 57 308
2 41 52
6 69 26
1 50 235
6 50 187
1 43 192
9 12 506
7 30 258
0 2 585
4 15 80
0 44 151
8 62 555
0 42 374
5 45 441
1 62 588
0 18 3
2 56 31
6 28 676
8 51 120
0 35 335
3 62 286
0 7 597
5 30 580
7 65 171
0 65 620
2 66 35
7 40 703
2 54 293
8 65 88
0 15 17
7 42 251
7 27 435
2 18 91
2 66 382
3 35 495
4 2 593
1 31 443
2 22 167
0 66 220
4 48 644
1 4 500
0 50 159
0 3 560
5 51 464
8 62 623
3 27 175
2 45 225
6 27 480
8 36 124
3 41 335
3 3 392
0 7 241
9 50 327
8 6 468
3 52 255
9 51 540
2 39 634
5 70 402
4 65 132
8 51 433
5 52 667
1 39 451
8 13 623
5 9 597
7 40 171
9 6 183
4 66 421
6 1 638
0 33 572
4 11 41
0 71 154
4 41 253
8 18 94
1 60 51
6 6 309
2 18 251
5 4 265
0 10 639
6 34 684
8 22 332
6 58 591
1 21 115
3 36 598
9 59 248
8 3 430
8 10 212
1 66 90
3 23 681
1 65 308
4 23 490
0 71 709
6 45 366
8 9 506
5 22 267
5 37 562
4 66 568
5 32 448
2 33 619
0 4 471
0 5 262
8 41 412
7 42 487
0 9 560
9 23 90
6 66 7
6 65 605
8 64 686
3 67 210
9 66 271
2 42 472
6 47 699
6 7 197
9 22 320
7 1 94
2 40 584
5 68 489
0 58 709
0 66 272
1 25 510
1 36 375
1 9 516
2 17 184
4 37 470
4 51 52
2 20 105
1 33 660
7 27 149
2 49 322
8 49 107
6 65 457
1 49 453
0 22 491
9 11 328
2 70 326
8 14 572
0 35 341
5 31 614
3 11 128
0 4 72
8 40 518
1 71 82
4 65 117
2 34 66
7 2 258
2 3 573
2 22 643
9 49 438
2 67 126
4 8 550
4 34 222
2 50 611
0 13 201
2 58 43
0 29 628
5 68 498
2 18 28
1 51 458
2 1 585
7 23 692
8 71 594
6 33 368
6 1 421
3 50 66
5 45 158
2 36 541
4 19 345
6 71 156
7 57 340
3 42 421
5 29 546
8 66 578
6 29 242
0 5 172
6 35 426
7 65 649
8 3 321
6 13 451
1 18 225
0 10 618
5 43 342
8 70 432
