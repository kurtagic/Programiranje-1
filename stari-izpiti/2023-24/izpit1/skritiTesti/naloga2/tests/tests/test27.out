[10, 7, 8, 4, 2, 9, 11, 8, 5, 4]
